layouts/shortcodes/pages/connect.zh.html

```diff
@@ -0,0 +1,123 @@
+<h2 class="text-center heading-line"><span>邮件列表</span></h2>
```

Comment:
See above comment. If we use i18n files, the management of shortcodes, and changes made to them become way easier!

Reply:
I think I can have a try.

Reply:
I've started work on this, don't worry about it. I'll have a version up in the next hour or so. I had some meetings that delayed me having a version finished

Reply:
That will be fantastic, to be honest, I am not so familiar with the it. and I really afraid that I will make things mess.^-^